packages/wonder-blocks-dropdown/components/option-item.js

```diff
@@ -139,12 +132,14 @@ const {blue, white, offBlack, offBlack32} = Color;
 
 const styles = StyleSheet.create({
     itemContainer: {
+        display: "flex",
```

Comment:
Is this necessary? Don't all `View`s have this by default?

Reply:
At some point it was a `addStyle("button")` but now it's a `View`, so I can remove it!